src/pyrokinetics/pyro.py

```diff
@@ -200,6 +200,12 @@ def __init__(
         # Load global kinetics file if it exists
         if kinetics_kwargs is None:
             kinetics_kwargs = {}
+        if kinetics_type == "IMAS":
+            if not hasattr(self, "eq"):
+                raise ValueError(
+                    "Loading a Kinetics object from IMAS requires an Equilibrium to be loaded"
+                )
+            kinetics_kwargs["eq"] = self.eq
```

Comment:
This feels like a bit of a smell to me, but I might need to think about it a bit more. The two things I'm not keen on:

1. can we be more specific than needing the whole equilibrium? Maybe just pass `psi` (or whatever is needed)? Or maybe this is feedback for upstream, that it would be better for this information to be stored in the file? Could this be something we _always_ pass?
2. hardcoding "IMAS" here -- what if we have other formats that need this information? Maybe this can be a property of the kinetics types so we can more generically check this?

Basically, this feels necessary but inelegant -- what are our constraints and options here?

Reply:
I think this could be removed entirely as this will get caught later on by the error of not having a `Equilibrium`

Reply:
Removed